### ISSUE TITLE
Forward pass of the model - how to pass videos?

### ISSUE THREAD

**Issue by esh04** [DAMO-NLP-SG/VideoLLaMA2#105]
I see this is how you run inference on the model:

```
    output = model.generate(
              input_ids, 
              return_dict_in_generate=True,
              output_scores=True,
              images_or_videos=videos,
              modal_list=['video'],
              max_new_tokens=128,
              do_sample=True,
              temperature=0.2,
              use_cache=False,
          )
```

I wanted to know how to run training code. The forward pass of the model does not support `modal_list` and `images_or_videos` - then how do I pass the videos? I want to be able to pass `labels` as I need to calculate probabilities. (so I need logits)
